deploy/examples/data-index.yaml

```diff
@@ -24,19 +24,3 @@ spec:
   infinispan:
     # will set this instance to auto provision an infra structure in the namespace and attach Infinispan credentials to it
     useKogitoInfra: true
-    # name of the auth realm. "default" is the realm name for
```

Comment:
I didn't understand your question :disappointed:

You mean the comment there? Oh, I just removed to not make ppl confused with a bunch of comments in the yaml since the `useKogitoInfra` is the most important. The full docs can be viewed in the CRD file. :)

Reply:
alright.